editor.planx.uk/src/pages/FlowEditor/lib/analytics/utils.ts

```diff
@@ -29,7 +29,10 @@ export function extractNodeTitle(node: Store.Node): string {
   const nodeTitle =
     node?.type === TYPES.Content
       ? getContentTitle(node)
-      : node?.data?.title ?? node?.data?.text ?? node?.data?.flagSet;
+      : node?.data?.title ??
+        node?.data?.text ??
+        node?.data?.flagSet ??
+        node?.data?.category;
```

Comment:
This is a small un-related change to ensure `Filter` node types capture `node_title` as their flagset category name, rather than undefined like now